mongo/options/changestreamoptions.go

```diff
@@ -46,6 +46,10 @@ type ChangeStreamOptions struct {
 	// corresponding to an oplog entry immediately after the specified token will be returned. If this is specified,
 	// ResumeAfter and StartAtOperationTime must not be set. This option is only valid for MongoDB versions >= 4.1.1.
 	StartAfter interface{}
+
+	// Custom options to be added to the initial aggregate for the change stream. Key-value pairs should correlate with
+	// desired option names and values. Values must be Marshalable.
```

Comment:
Suggest noting that non-custom options (or perhaps "named options") should be preferred when possible. Perhaps this:

```suggestion
	// desired option names and values. Values must be Marshalable. Custom options may conflict with non-custom options.
	// Custom options bypass client-side validation. Prefer using non-custom options when possible.
```

Reply:
Added.